internal/changestream/stream/package_test.go

```diff
@@ -81,12 +84,16 @@ func (s *baseSuite) expectAfter() chan<- time.Time {
 	return ch
 }
 
-func (s *baseSuite) expectAfterAnyTimes() {
+func (s *baseSuite) expectAfterAnyTimes(done chan struct{}) {
 	s.clock.EXPECT().After(defaultWaitTermTimeout).Return(make(chan time.Time)).AnyTimes()
 	s.clock.EXPECT().After(gomock.Any()).DoAndReturn(func(d time.Duration) <-chan time.Time {
```

Comment:
How do you know which of these you are matching?

If they are exclusive, you should write a matcher that excludes `defaultWaitTermTimeout`, rather than using `Any()`.

Reply:
This is the back-off strategy, I'll extract it out to indicate that.